src/components/Chips/Chips.module.scss

```diff
@@ -18,9 +18,7 @@
   animation: chips-pop-in-emphasized var(--motion-productive-medium) var(--motion-timing-emphasize);
   @include font-general-text();
   &.border {
-    height: 22px;
-    border: 1px solid var(--text-color-on-primary);
-    line-height: 21px;
+    border: 1px solid var(--primary-background-color);
```

Comment:
I think `primary-background-color` is a mistake here. Have Meirav told you to use this one?
<img width="724" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/2fdc35ce-58aa-4fd9-8761-540cb4c7719f">
<img width="589" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/0a4297f6-8a83-4518-bf14-81c9bd8b3f78">

You know how to switch themes in Storybook? It's this button
<img width="225" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/ee10c93b-7981-4127-94f3-d37ab8da740f">